css/css-display/accessibility/display-contents-role-and-label.html

```diff
@@ -0,0 +1,290 @@
+        <td>x</td>
+      </tr>
+    </tbody>
+  </table>
+
+  <!-- Display: Grid -->
+  <div style="display: grid; grid-template-columns: 1fr 100px; grid-gap: 1em;">
+      <h2>x</h2>
+      <ul style="display: contents" data-expectedrole="list" data-testname="ul with display: contents, as child of div with display: grid, has list role" class="ex-role">
+          <li>x</li>
+          <li data-expectedrole="listitem" data-expectedlabel="x" data-testname="listitem within ul with display: contents, as child of div with display: grid, has listitem role" class="ex-role-and-label">x</li>
+      </ul>
+  </div>
+
+  <!-- Generics -->
+  <div style="display: contents;" data-testname="div with display: contents has generic role" class="ex-generic">x</div>
+  <span style="display: contents;" data-testname="span with display: contents has generic role" class="ex-generic">x</span>
+
+  <!-- Headings -->
+  <h3 style="display: contents;" data-expectedrole="heading" data-expectedlabel="x" data-testname="h3 with display: contents has heading role" class="ex-role-and-label">x</h3>
+  <div role="heading" aria-level="3" style="display: contents;" data-expectedrole="heading" data-expectedlabel="x" data-testname="div with heading role, aria-level=3 and display: contents has heading role" class="ex-role-and-label">x</div>
+  <span role="heading" aria-level="1" style="display: contents;" data-expectedrole="heading" data-expectedlabel="x" data-testname="span with heading role, aria-level=1 and display: contents has heading role" class="ex-role-and-label">x</span>
+
+  <!-- Landmarks and regions -->
+  <header style="display: contents;" data-expectedrole="banner" data-testname="header with display: contents has banner role" class="ex-role">x</header>
+  <nav style="display: contents;" data-expectedrole="navigation" data-testname="nav with display: contents has navigation role" class="ex-role">x</nav>
+  <aside style="display: contents;" data-expectedrole="complementary" data-testname="aside with display: contents has complementary role" class="ex-role">x</aside>
+  <main style="display: contents;" data-expectedrole="main" data-testname="main with display: contents has main role" class="ex-role">x</main>
+  <footer style="display: contents;" data-expectedrole="contentinfo" data-testname="footer with display: contents has contentinfo role" class="ex-role">x</footer>
+  <form aria-label="label" style="display: contents;" data-expectedrole="form" data-expectedlabel="label" data-testname="form with display: contents has form role" class="ex-role-and-label">x</form>
+  <search style="display: contents;" data-expectedrole="search" data-testname="search with display: contents has search role" class="ex-role">x</search>
+  <section aria-label="label" style="display: contents;" data-expectedrole="region" data-expectedlabel="label" data-testname="section with aria-label and display: contents has region role" class="ex-role-and-label"></section>
```

Comment:
Fine as-is, but it wasn't immediately apparent why most test role, and a few test role and label?

Reply:
Good catch, I missed these. I'll add role-and-label tests to applicable tests; I also need to add `aria-label` tests for nav/aside.

Reply:
Fixed, I believe: e9a79c5ff0a99b54ccd0011f3f6a8b663908bfc1.